dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/GitHubSourceCodeRepo.java

```diff
@@ -326,9 +326,7 @@ public Service initializeServiceFromGitHub(String repositoryId, String subclass)
                 descriptorLanguageSubclass = DescriptorLanguageSubclass.convertShortNameStringToEnum(subclass);
             } catch (UnsupportedOperationException ex) {
                 // TODO: https://github.com/dockstore/dockstore/issues/3239
-                String msg = "Subclass " + subclass + " is not a valid descriptor language subclass.";
-                LOG.info(msg);
```

Comment:
Might be still worth a `LOG.debug` for when we're ... debugging

Reply:
The messaged were being duplicated since they were printed here but also printed in AbstractWorkflowResource.java line 321. 

Now they are only printed at line 321.

Reply:
They will be LOG.info though, not LOG.debug.